List systems from db and return via API

Josca opened the pull request:
SPM-27

semtexzv commented on manager/controllers/paging.go:
```diff
@@ -0,0 +1,66 @@
+
+import "fmt"
+
+func CreateLinks(path string, offset, limit, total int, dataFormat string, showAll bool) Links {
```
What's the reason for show_all parameter ?

Josca replied:
to show all systems or only for current account

jiridostal replied:
It should allow you to show all Advisories, not only those that are affecting your environment. Currently it's not a requirement so we can remove it.

Josca commented:
@jiridostal @semtexzv I've removed `show_all` param and added generic `otherParams` param to enable inserting of additional query params co `CreateLinks` method.
